chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/SurveillanceDAOImpl.java

```diff
@@ -716,14 +769,14 @@ private SurveillanceEntity fetchSurveillanceById(Long id) throws EntityRetrieval
                         + "LEFT JOIN FETCH cce2.certificationEdition "
                         + "LEFT OUTER JOIN FETCH ncs.nonconformityStatus " + "LEFT OUTER JOIN FETCH ncs.documents "
                         + "WHERE surv.deleted <> true " + "AND surv.certifiedProductId = :cpId",
-                SurveillanceEntity.class);
+                        SurveillanceEntity.class);
         query.setParameter("cpId", id);
 
         List<SurveillanceEntity> results = query.getResultList();
         return results;
     }
 
-    private PendingSurveillanceEntity fetchPendingSurveillanceById(Long id, boolean includeDeleted)
+    private PendingSurveillanceEntity fetchPendingSurveillanceById(final Long id, final boolean includeDeleted)
```

Comment:
When do we "includeDeleted" pending surveillances? Why do we? I know this isn't really in your SOW, but this seems weird

Reply:
Addressed.